system/emergency_goal_manager/include/emergency_goal_manager_core.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef EMERGENCY_GOAL_MANAGER_CORE_HPP_
+#define EMERGENCY_GOAL_MANAGER_CORE_HPP_
+
+// Autoware
```

Comment:
Comments in include sections disable sorting, so avoid them if you don't need them.
https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/languages/cpp/#include-header-files-in-the-defined-order-required-partially-automated

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/5634/commits/ee8146642a3865bb2c5550060cd41a8bff43f262